app/components/ak-button/index.scss

```diff
@@ -0,0 +1,29 @@
+.btn {
+  height: 2.5rem;
+  padding: 0.57em 0.85em;
+  border: none;
+  border-radius: var(--button-border-radius);
+  font-size: 1rem;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: center;
+  column-gap: 0.5em;
+  line-height: 2.5rem;
```

Comment:
are these to even required?

Reply:
yes, it required to be overwrite with global style which is 24px set to button element. which is ideally required to align the inner content at center by pos.